torch/_refs/__init__.py

```diff
@@ -520,6 +520,50 @@ def log10(a):
     return prims.log10(a)
 
 
+@out_wrapper
+def log_softmax(
+    a: TensorLikeType,
+    dim: int,
+    *,
+    dtype: Optional[torch.dtype] = None,
+) -> TensorLikeType:
+    result_dtype = dtype or a.dtype
+    computation_dtype = utils.get_computation_dtype(a.dtype)
+    a = prims.convert_element_type(a, computation_dtype)
+    a_max = amax(a, dim, keepdim=True)
+    shifted = a - a_max
+    shifted_logsumexp = logsumexp(shifted, dim, keepdim=True)
+    return prims.convert_element_type(
+        shifted - shifted_logsumexp, result_dtype
+    )
+
+
+def _squeeze_multiple(a: TensorLikeType, dims: DimsSequenceType) -> TensorLikeType:
```

Comment:
Unify this with prims.squeeze -- see 

https://github.com/pytorch/pytorch/blob/38e717dc876e0f9af66b32e9cc59b44b1f9edf27/torch/_prims/__init__.py#L1636

I think we only need one? But maybe this implementation is better for the prim?

Reply:
I didn't notice that `prims.squeeze` works for multiple specified dimensions. Both implementations are fine, I will not touch `prims.squeeze` at this time.